plugins/action/configure.py

```diff
@@ -117,7 +117,8 @@ def run(self, tmp=None, task_vars=None):
             return self._result
         result = super(ActionModule, self).run(tmp, task_vars)
 
-        json_config = json.loads(self._task.args.get("config") or {})
+        json_config = json.dumps(self._task.args.get("config"))
```

Comment:
This is not required and json data can be directly passed to `json_to_xml` method after PR https://github.com/ansible-collections/community.yang/pull/36 is merged